index.html

```diff
@@ -1507,11 +1589,13 @@ <h3>Credential Subject</h3>
         <pre class="example nohighlight" title="Usage of the credentialSubject property">
 {
   "@context": [
-    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/v2",
     "https://www.w3.org/2018/credentials/examples/v1"
```

Comment:
```suggestion
    "https://www.w3.org/2021/credentials/examples/v2"
```

I'm going to stop these suggestions here since most of the rest of the changeset has to do w/ changing these two lines. A simple search/replace should do the trick. Please make the appropriate changes @tplooker .